packages/client/src/events/events.ts

```diff
@@ -0,0 +1,4 @@
+import { ClientProviderEvents } from '@openfeature/core';
+
+export { ClientProviderEvents as ProviderEvents};
+export type ProviderEmittableEvents = Exclude<ClientProviderEvents, ClientProviderEvents.ContextChanged>;
```

Comment:
Please add a JSDoc that explains why we're using a subset of ClientProviderEvents.

Reply:
done